tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -44,7 +44,10 @@ export const deleteUser = async ({ user, admin }: { user: User; admin: User }):
     path: join('graph', 'v1.0', 'users', user.id),
     user: admin
   })
-
+  try {
+    const usersEnvironment = new UsersEnvironment()
+    usersEnvironment.removeUser({ key: user.id })
+  } catch (e) {}
```

Comment:
why try catch?

Reply:
It will be removed in the next PR. I just want to save time so tried so fast fix-up here.

Reply:
Not a nice way to do though